source/dotnet/Library/AdaptiveCards/Rendering/RendererUtilities.cs

```diff
@@ -13,8 +13,8 @@ namespace AdaptiveCards.Rendering
     public static class RendererUtilities
     {
         private static readonly Regex TextFunctionRegex =
-            new Regex(@"\{\{(?<func>DATE|TIME){1}\((?<date>.+?){1}(?:,\s*(?<hint>Short|Long){1}\s*)??\)\}\}",
-                RegexOptions.ExplicitCapture | RegexOptions.IgnoreCase);
+            new Regex(@"\{\{(?<func>DATE|TIME){1}\((?<date>\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d[Z+-].*?){1}(?:,\s*(?<hint>SHORT|LONG|COMPACT){1}\s*)??\)\}\}",
```

Comment:
This regex doesn't look quite right to me -- for the date, shouldn't we have something like this?
```regex
(?<date>[\d]{4}-[\d]{2}-[\d]{2}T[\d]{2}:[\d]{2}:[\d]{2}[Z+-].*?){1}
```

Reply:
Both are the same exact regex. Not sure which is more clear or preferred though.

Reply:
I think the counted character class version is a bit more intuitive to a reader -- you don't have to count the digits :)